src/main/java/com/project/momo/service/MemberService.java

```diff
@@ -0,0 +1,112 @@
+package com.project.momo.service;
+
+import com.project.momo.common.utils.AuthUtils;
+import com.project.momo.dto.member.MemberInfoResponse;
+import com.project.momo.dto.member.PasswordUpdateRequest;
+import com.project.momo.dto.payment.PaymentRequest;
+import com.project.momo.entity.Member;
+import com.project.momo.entity.Payment;
+import com.project.momo.repository.MemberRepository;
+import com.project.momo.repository.PaymentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+    private final PaymentRepository paymentRepository;
+    private final PasswordEncoder passwordEncoder;
+
+    private final short MAX_PAYMENT_CNT = 3;
```

Comment:
```suggestion
    private static final short MAX_PAYMENT_CNT = 3;
```
상수는 static을 달아서 사용합니다 :)